packages/react-reconciler/src/ReactFiberReconciler.new.js

```diff
@@ -714,6 +717,11 @@ export function getIsStrictModeForDevtools() {
 }
 
 export function setIsStrictModeForDevtools(newIsStrictMode: boolean) {
+  // We're in a test. Disable yielding during the double render
```

Comment:
i don't totally follow this comment. why are we in a test here?

EDIT: oh nvm, you mean inside the if

Reply:
in any case, would you mind adding some context about why we need to do this here?